src/JWT/TokenExpiredException.cs

```diff
@@ -1,12 +1,38 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace JWT
 {
     public class TokenExpiredException : Exception
     {
-        public TokenExpiredException(string message)
+       private const string PayloadDataKey = "PayloadData";
+       private const string ExpirationKey = "Expiration";
+
+       public TokenExpiredException(string message)
             : base(message)
         {
         }
+
+        public Dictionary<string, object> PayloadData
+        {
+            get
+            {
+               if (Data.Contains(PayloadDataKey))
```

Comment:
I was debating whether to throw or not... The problem I have with throwing is that since you requested that `TokenExpiredException` inherits from `SignationVerificationException` (see PR #33), the `Received` and `Expected` properties will not be assigned for `TokenExpiredException`... so making a `catch (SignationVerificationException ex)` may crash if we don't do that...